nautobot/docs/additional-features/jobhooks.md

```diff
@@ -0,0 +1,21 @@
+# Jobhooks
```

Comment:
As @jathanism pointed out, we should probably use `job hook` in the documentation and UI rather than `jobhook` as this isn't a standard term of art like `webhook` is.